### ISSUE TITLE
Data not adjusted for splits/dividend

### ISSUE THREAD

**Issue by ven7782** [pipegalera/intraday_data#1]
I was planning to use the 1 Minute S&P500 intraday data for NVDA. However I noticed that the data is not adjusted for splits and dividend. For E.G. On 2024-01-02 NVDA opened at 493.74 but adjusted for splits/dividend it should be 49.24. Because of this, the data is pretty much useless. It will only work for stocks which have not been split. If you wish, I can submit a PR for adjusting the price for splits/dividend.

**Comment by pipegalera** [comment]
Thanks @ven7782 for raising the issue. I'm updating all the historical data adjusted by splits and dividends. It will take a while since it's 10 min per ticker and there are 503 tickers. Check next Monday and let me know.

**Comment by pipegalera** [comment]
@ven7782 I updated NVDA ticker. Feel free to check !

**Comment by ven7782** [comment]
Will do Thanks for updating
